consensus/src/checkpoint.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::common::{Chain, ProtocolStateChain};
+use mina_crypto::base58::Base58Encodable;
+use mina_crypto::hash::{EpochSeed, StateHash};
+use mina_rs_base::protocol_state::ProtocolState;
+#[allow(dead_code)]
+const SLOTS_PER_EPOCH: u32 = 7140;
+
+#[allow(dead_code)]
+fn init_checkpoints(genesis: &mut ProtocolState) {
+    genesis.body.consensus_state.staking_epoch_data.seed = EpochSeed::default();
+    genesis
+        .body
+        .consensus_state
+        .staking_epoch_data
+        .start_checkpoint = StateHash::default();
+    genesis
+        .body
+        .consensus_state
+        .staking_epoch_data
+        .lock_checkpoint = StateHash::default();
+    genesis
+        .body
+        .consensus_state
+        .staking_epoch_data
+        .epoch_length
+        .0 = 1;
+    genesis.body.consensus_state.next_epoch_data.seed =
+        Base58Encodable::from_base58("2vaRh7FQ5wSzmpFReF9gcRKjv48CcJvHs25aqb3SSZiPgHQBy5Dt")
+            .unwrap();
```

Comment:
We should not call `.unwrap` in non-test code, either return Result or use expect